Demo/Demo/SwiftUIComponents/SwiftUINativeCheckoutDemo.swift

```diff
@@ -18,6 +18,10 @@ struct SwiftUINativeCheckoutDemo: View {
 
     @State var checkoutTypeSelection = CheckoutType.order
 
+    init() {
+        UITableView.appearance().tableHeaderView = UIView(frame: CGRect(x: 0, y: 0, width: 0, height: Double.leastNonzeroMagnitude))
+        UITableView.appearance().tableFooterView = UIView(frame: CGRect(x: 0, y: 0, width: 0, height: Double.leastNonzeroMagnitude))
```

Comment:
What is this for? Trying to hide the default header and footer view?

Reply:
its a hacky way of reducing the space the `Form` takes. Honestly this is just taken from stackoverflow, but we can pair on this to see how it works.